Casks/google-chrome.rb

```diff
@@ -9,7 +9,7 @@
 
   livecheck do
     url "https://chromiumdash.appspot.com/fetch_releases?channel=Stable&platform=Mac"
-    regex(/"version": "(\d+(?:\.\d+)+)"/i)
+    regex(/"version":"(\d+(?:\.\d+)+)"/i)
```

Comment:
This seems like an easy thing to change upstream, how about
```suggestion
    regex(/"version": ?"(\d+(?:\.\d+)+)"/i)
```

Reply:
Good point. Since you can do all sorts of whitespace shenanigans and still have valid JSON, I thought it's even better to switch `livecheck` over to do JSON parsing.